manifests/jsonnetfile.json

```diff
@@ -10,6 +10,16 @@
       },
       "version": "v1.14.3",
       "name": "kube-libsonnet"
+    },
+    {
+      "source": {
+        "git": {
+          "remote": "https://github.com/bitnami-labs/kubernetes-grafana-dashboards",
+          "subdir": ""
+        }
+      },
+      "version": "jjo-fix-kube-1.16-pod_container-labels",
```

Comment:
@dbarranco @jbianquetti-nami will obviously fix to an specific commit, 
once bitnami-labs/kubernetes-grafana-dashboards#15 is merged.

Reply:
^ done (lock kubernetes-grafana-dashboards to kubernetes-grafana-dashboards#15 merge commit)